src/backoffice/dashboards/examine-management/views/section-view-examine-searchers.ts

```diff
@@ -0,0 +1,306 @@
+import { UUITextStyles } from '@umbraco-ui/uui-css/lib';
+import { css, html, LitElement } from 'lit';
+import { customElement, state, query, property } from 'lit/decorators.js';
+
+import { UmbModalService } from '../../../../core/services/modal';
+import { UmbNotificationService } from '../../../../core/services/notification';
+import { UmbNotificationDefaultData } from '../../../../core/services/notification/layouts/default';
+
+import { UmbContextConsumerMixin } from '@umbraco-cms/context-api';
+
+import { ApiError, ProblemDetails, SearchResult, SearchResource, Field } from '@umbraco-cms/backend-api';
+
+import '../../../../core/services/modal/layouts/fields-viewer/fields-viewer.element';
+import '../../../../core/services/modal/layouts/fields-viewer/fields-settings.element';
```

Comment:
Let's make a shortcut to import from the services folder, just like we did for `@umbraco-cms/backend-api` and others. I believe they are made in the `tsconfig.json` file.